iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -292,9 +295,28 @@ def _adjust_operator_bundle(
                     bundle_metadata, org_enclose_repo_namespace, org_enclose_repo_glue
                 )
         elif customization_type == 'related_bundles':
+            # When recursive_related_bundles is set to True, the call is from a
+            # recureive_related_bundles request. Product teams have customizations
+            # in the order so that when this customization is specified, the bundles
+            # are accessible. For the recursive_related_bundles request, we want the
+            # the images to be in an accessible state. So apply all customizations
+            # that were specified before this one and return.
+            if recursive_related_bundles:
+                log.debug(
+                    'related_bundles customization found with recursive_related_bundles set'
+                    ' to True. No further customizations will be performed for organization'
+                    f' {organization}. Finding recursive_related_bundles'
```

Comment:
Combined with the comment above, maybe a wording like

```
Stopping before performing `related_bundles` modification since `<varname>` is set to 
True. No further customizations will be performed as specified for organization {organization}.
```